src/etc/inc/services.inc

```diff
@@ -950,6 +950,7 @@ EOD;
                 $newzone['dns-servers'] = array($dhcpifconf['ddnsdomainprimary']);
                 $newzone['ddnsdomainkeyname'] = $dhcpifconf['ddnsdomainkeyname'];
                 $newzone['ddnsdomainkey'] = $dhcpifconf['ddnsdomainkey'];
+                $newzone['ddnsdomainalgorithm'] = $dhcpifconf['ddnsdomainalgorithm'];
```

Comment:
@itn3rd77 you need to set the default, like:

`$newzone['ddnsdomainalgorithm'] = !empty($dhcpifconf['ddnsdomainalgorithm']) ? $dhcpifconf['ddnsdomainalgorithm'] : "hmac-md5";
`

Otherwise you will break running configs.

Reply:
fixed. thanks for catching.